tests/test_gae.py

```diff
@@ -25,18 +25,22 @@ def reset(self, seed: Optional[int] = None):
         if seed is not None:
             self.observation_space.seed(seed)
         self.n_steps = 0
-        return self.observation_space.sample()
+        return self.observation_space.sample(), {}
 
     def step(self, action):
         self.n_steps += 1
 
-        done = False
+        done = truncated = False
         reward = 0.0
         if self.n_steps >= self.max_steps:
             reward = 1.0
             done = True
+            # To simplify GAE computation checks,
```

Comment:
but does the GAE calculation distinguish between truncation and termination? from here https://github.com/tlpss/stable-baselines3/blob/95bb4d6b880f21e8e3117d29d4391937f12fedbf/tests/test_gae.py#L96 it seems not? (fixing this might be not in the scope of this PR, but I feel the GAE tests should cover this now that the truncation is  added)

Reply:
See test below that one, we handle truncations like that: #658 
It is a trick that allows to keep the same code when truncation happens.